now we have a real problem:

```
[  5%] Building CXX object monitored_vcs/info_terminal/infremen/CMakeFiles/infremen.dir/src/infremen.cpp.o
/var/lib/jenkins/workspace/pr-indigo-aaf_deployment/monitored_vcs/info_terminal/infremen/src/infremen.cpp:22:30: fatal error: info_task/Clicks.h: No such file or directory
 #include <info_task/Clicks.h>
                              ^
```

--- a/info_terminal/infremen/CMakeLists.txt
+++ b/info_terminal/infremen/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(infremen)
 
-find_package(catkin REQUIRED COMPONENTS  mongodb_store message_generation roscpp strands_executive_msgs std_msgs std_srvs strands_navigation_msgs mongodb_store_msgs geometry_msgs dynamic_reconfigure) 
+find_package(catkin REQUIRED COMPONENTS  mongodb_store message_generation roscpp strands_executive_msgs std_msgs std_srvs strands_navigation_msgs mongodb_store_msgs geometry_msgs dynamic_reconfigure info_task) 
 
 ## Generate messages in the 'msg' folder
 add_message_files(
@@ -22,7 +22,7 @@ generate_dynamic_reconfigure_options(
 
 set ( CMAKE_CXX_FLAGS "-Ofast -march=native -mno-avx -ggdb")
 
-catkin_package(CATKIN_DEPENDS roscpp strands_executive_msgs message_generation std_msgs std_srvs strands_navigation_msgs mongodb_store_msgs geometry_msgs mongodb_store)
+catkin_package(CATKIN_DEPENDS roscpp strands_executive_msgs message_generation std_msgs std_srvs strands_navigation_msgs mongodb_store_msgs geometry_msgs mongodb_store info_task)
 
 include_directories(${catkin_INCLUDE_DIRS}) 
 

--- a/info_terminal/infremen/CMakeLists.txt
+++ b/info_terminal/infremen/CMakeLists.txt
@@ -28,7 +28,7 @@ include_directories(${catkin_INCLUDE_DIRS})
 
 add_executable(infremen src/infremen.cpp)
 
-add_dependencies(infremen ${catkin_EXPORTED_TARGETS} mongodb_store ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_genmsg)
+add_dependencies(infremen ${catkin_EXPORTED_TARGETS} mongodb_store ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_genmsg infremen_generate_messages)
 
 add_library(infremen_frelement src/CFrelement.cpp)
 add_library(infremen_timer src/CTimer.cpp)

--- a/info_terminal/infremen/package.xml
+++ b/info_terminal/infremen/package.xml
@@ -48,6 +48,7 @@
   <build_depend>strands_executive_msgs</build_depend>
   <build_depend>mongodb_store_msgs</build_depend>
   <build_depend>scitos_msgs</build_depend>
+  <build_depend>info_task</build_depend>
   
   <run_depend>roscpp</run_depend>
   <run_depend>genmsg</run_depend>
@@ -61,6 +62,7 @@
   <run_depend>mongodb_store_msgs</run_depend>
   <run_depend>geometry_msgs</run_depend>
   <run_depend>scitos_msgs</run_depend>
+  <run_depend>info_task</run_depend>
 
   <!-- The export tag contains other, unspecified, tags -->
   <export>

